class Class_Errors:
    def __init__(self):
        return


